src/components/Layout.js

```diff
@@ -79,10 +79,10 @@ type Props = {
 const Layout = ({ container, children, ...props }: Props) => (
   <ThemeProvider theme={bsTheme}>
     <PageRoot>
-      <Footer />
       <ContentRoot>
         {container ? <MainContainer {...props}>{children}</MainContainer> : children}
       </ContentRoot>
+      <Footer />
```

Comment:
I had the footer before the content so that the content would appear on top of it. Does it look ok with the footer on top (of the cards)?

Reply:
fixed it